microsoft-edge/webview2/how-to/javascript.md

```diff
@@ -30,6 +30,24 @@ Use the following functions to begin embedding JavaScript in your WebView app.
 | [ExecuteScriptAsync](/dotnet/api/microsoft.web.webview2.wpf.webview2.executescriptasync) | Run JavaScript in a WebView control. For more information, navigate to the Get Started tutorial. |
 | [OnDocumentCreatedAsync](/microsoft-edge/webview2/reference/win32/icorewebview2#addscripttoexecuteondocumentcreated) | Runs when the Document Object Model (DOM) is created. |
 
+<!-- ====================================================================== -->
+## Scenario: ExecuteScript JSON Encoded Results
+
+Because the result of is ExecuteScriptAsync is JSON encoded, if the result evaluating the JavaScript is a string, then you will receive a JSON encoded string and not the value of the string. For example the following script will result in a string with the following value including the quotes at the start and end, and the escaping slashes. 
+ * Script: "example \" \' \\ "; 
```

Comment:
The "rich view" of this is losing the backslashes. I think this needs to be within a code block to show up properly.

Reply:
@vchapel - This seems like a tagging fix for you to implement.  Probably just wrap with backticks, not sure if remove escapes or keep them.  See the rendered page.  https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/javascript?branch=pr-en-us-1636#scenario-executescript-json-encoded-results
You could look at the branch/PR in VS Code, and try its .md Preview view.

Reply:
@champnic adjusted the tagging to show backslashes in a code block. Is this correct?

![image](https://user-images.githubusercontent.com/84938940/148126450-218bc275-0d17-4ed5-9f31-6a23af1d1699.png)

Reply:
@champnic I placed the backslash section in code blocks, is this correct?

![image](https://user-images.githubusercontent.com/84938940/148126753-efd04968-daad-40c6-8900-1f00adbe7cf4.png)

Reply:
The formatting looks correct now. However, I'm wondering if the values are correct. @jasonstephen15 Where did we get this example? If it was JSON encoded I would expect a bunch more backslashes in the result no? From playing around with this I get:
```cs
var result = await webView22.CoreWebView2.ExecuteScriptAsync(@"'example \""';");
result == "\"example \\\"\""
```

I wonder if a simpler example would demonstrate the encoding more clearly and still be helpful:
```cs
var result = await webView22.CoreWebView2.ExecuteScriptAsync(@"'example'");
result == "\"example\"";
```
Where the quotations around the string have been encoded by JSON.

Reply:
From me 😊. The script & result in the above are the raw values of the string not represented with encoding. Using a code snippet like you have above should be more explicit about encoding / escaping